packages/components/src/functional-components/FormField/FormField.tsx

```diff
@@ -124,11 +124,7 @@ const KolFormFieldFc: FC<FormFieldProps> = (props, children) => {
 	}
 
 	return (
-		<Component
-			class={clsx('kol-form-field', stateCssClasses, classNames)}
-			role={`presentation` /* Avoid element being read as 'clickable' in NVDA */}
```

Comment:
Die role=presentation wurde hier wohl irrtümlich eingefügt. Die hatten wir zuvor nur auf dem _Host_-Element `kol-input`, das es in der Form jetzt nicht mehr gibt.

Das role=presentation hatte in Chrome zu dem etwas eigenartigen Verhalten geführt, dass _alle_ Radio-Buttons auf der Seite, unabhängig von ihrem `name`-Attribut für die "1 von {n}"-Ausgabe gezählt wurden.